### PR TITLE
chore: Updating Azure NPM to v1.0.33

### PR DESCRIPTION
**Pull request by jaer-tsun**
**Reason for Change**:

Contains the latest changes in the v1.0.33 Azure NPM release
https://github.com/Azure/azure-container-networking/releases/tag/v1.0.33

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/install-dependencies.sh [hunk 1]
```diff
@@ -270,6 +270,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.0.33
```
Comment (CecileRobertMichon) [inline vhd/packer/install-dependencies.sh]: does NPM 1.0.33 require Azure CNI 1.0.33?

Comment (jaer-tsun) [reply vhd/packer/install-dependencies.sh]: nope, they're not related

I think Patrick updated CNI to v1.0.33, but not sure whether he did it for both windows and linux

**Pull request merged**